include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2849,7 +2857,9 @@ int bt_le_filter_accept_list_clear(void);
 /**
  * @brief Set (LE) channel map.
  *
- * @param chan_map Channel map.
+ * Used to inform the Controller of which RF channels to use for LE communication with the Host.
```

Comment:
I think there is typo changing the meaning completely :).

```suggestion
 * Used to inform the Controller of which RF channels to use for Bluetooth LE communication. Only lower 37 bits are valid.
```

Reply:
Changed the text up a bit now!